recipes/default.rb

```diff
@@ -12,7 +12,7 @@
 end
 
 # Command to configure w32time service
-manualpeerlist = "#{node['w32time']['ntpserver']},#{node['w32time']['flag']}"
+manualpeerlist = "#{node['w32time']['servers'].join(sep=",#{node['w32time']['flag']} ")},#{node['w32time']['flag']}"
```

Comment:
Because one flag will be missing at the end.
With the simple join:
ntp1,0x8 nt2,0x8 ntp3
But we need one more flag:
ntp1,0x8 nt2,0x8 ntp3,0x8